README.md

```diff
@@ -63,20 +63,20 @@ type ExampleConfig struct {
 // Defines default config option
 var (
     defaultConfig = ExampleConfig{
-		    Counter: 4,
+		    Counter: string(4),
```

Comment:
better keep `4` here and change the type of counter to `Int`.

Reply:
I see, I updated the pull-request with counter type `int`